src/components/OrderingPage/OrderingPart.tsx

```diff
@@ -0,0 +1,83 @@
+import './OrderingPart.scss';
+import { OrderingForm } from '../OrderingForm/OrderingForm';
+import { useState } from 'react';
+import x from './x.svg';
+
+export const OrderingPart = () => {
+  const [count, setCount] = useState(1);
+  const incrementCount = () => {
+    setCount(count + 1);
+  };
+
+  const [{ items }, setItems] = useState({ items: [<></>] });
+  const addItem = () => {
+    items.push(
+      <div key={items.length}>
+        <div className="form-inside">
+          <div className="product-number">
+            <div>
+              Məhsul # <span>{count}</span>
+            </div>
+            {count > 0 && (
+              <div className="close-order">
+                <button className="close-order-button">
+                  <img src={x} alt="" />
+                </button>
+              </div>
+            )}
+          </div>
+          <OrderingForm />
```

Comment:
are you sure that this component must be here?
in this case you will have several order forms on the page... for me, that's strange)

Reply:
you should do something with this code in this case))

Reply:
but it is working) I dont understand what is wrong with it